docs/packages/pkg/corrade.rst

```diff
@@ -0,0 +1,42 @@
+.. spelling::
+
+    corrade
+
+.. index::
+  single: graphics ; corrade
+
+.. _pkg.corrade:
+
+corrade
+===
+
+-  `Official <https://github.com/mosra/corrade>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/corrade/CMakeLists.txt>`__
+-  Developed by `Vladimír Vondruš<https://github.com/mosra>`
```

Comment:
Please change to
```
`Vladimír Vondruš <https://github.com/mosra>`__
```
(space and trailing `__`)

Reply:
Fixed by 432b6f4505ad83062a765c472ceef2ecfeb3ced0